README.md

```diff
@@ -2,33 +2,35 @@
 # How to Update the [MHKiT Website](https://MHKiT-Software.github.io/MHKiT/)
 
 ## Download/Install Required Packages
-### Step 1. Download and Install Python 
+### Step 1. Download and Install Python
   - Windows: Download and install Python using [Anaconda](https://www.anaconda.com/distribution/)
   - **NOTE:** MAC/LINUX skip to Step 3, Python is already installed and added to path
- 
+
 ### Step 2. Add Python to System Path
-  - Windows: Modify the PATH in environmental variables to include: C:\PYTHON;C:\PYTHON\Scripts 
+  - Windows: Modify the PATH in environmental variables to include: C:\PYTHON;C:\PYTHON\Scripts
   ([more info here](http://stackoverflow.com/questions/3701646/how-to-add-to-the-pythonpath-in-windows-7))
-  - **NOTE:** Anaconda skip to Step 3,  Python is already added to path 
+  - **NOTE:** Anaconda skip to Step 3,  Python is already added to path
   - **NOTE:** MAC/LINUX skip to Step 3, Python is already added to path
-  
+
```

Comment:
I want to combine these pages and simplify it to be in accordance with the documentation:
e.g. Remove steps 1&2 and put a requirements header which then states:
https://mhkit-software.github.io/MHKiT/installation.html including the MHKiT-Python and Requirements headers. 
Merge these as makes sense to you.

Reply:
@ssolson if we are asking people to use anaconda, the python path is automatically set, and this step can be removed. This is only necessary with certain python distributions.

Reply:
@kmruehl Your comment is outdated by the current PR. Please comment on the latest.

Reply:
@ssolson noted. I think the other comments still stand. I did not realize each of these threads were for a new revision to the PR.